tracer/test/Datadog.Trace.IntegrationTests/DiagnosticListeners/AspNetCoreDiagnosticObserverTests.cs

```diff
@@ -459,7 +459,7 @@ private static async Task AssertDiagnosticObserverSubmitsSpans(
                         break;
                     }
 
-                    Thread.Sleep(200);
+                    await Task.Delay(200).ConfigureAwait(false);
```

Comment:
oooh, I don't know about this one, xunit has a sync context, right?, I'm not sure if we should use this 🤔 
```suggestion
                    await Task.Delay(200);
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/7145/commits/140b6ac9b377428380005da150f82e712b168cc1